pkg/errors/errors.go

```diff
@@ -0,0 +1,18 @@
+package errors
+
+import (
+	"net/http"
+
+	errcode "github.com/docker/distribution/registry/api/errcode"
+)
+
+const errGroup = "openshift"
+
+var (
+	ErrorCodeFIXME = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
Having a dedicated error for certificate issues would be nice. Telling the certificate error from any other may not be that easy though.

Unless we know of any other unparse-able error, we could leave the rest of them unchanged and come back later when someone discovers one.

:+1: for this file. I bet there's gonna be more OpenShift specific errors.

Reply:
any objections to renaming it "ErrorCodeWrapper" ?